modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/form21a_spec.rb

```diff
@@ -56,6 +69,28 @@
       end
     end
 
+    context 'with mismatching schema' do
+      it 'logs the error and returns a bad request status' do
+        allow_any_instance_of(AccreditedRepresentativePortal::V0::Form21aController)
+          .to receive(:schema)
+          .and_return(schema)
```

Comment:
I think we could change this to an expectation/stub thing: "expect VetsJsonSchema to be called with 21A and return our test schema".